docs/static/includes/interfaces/bicep/int.diag.udt.schema.bicep

```diff
@@ -0,0 +1,35 @@
+
+// ============== //
+//   Parameters   //
+// ============== //
+
+import { diagnosticSettingFullType } from 'br/public:avm/utl/types/avm-common-types:0.4.0'
```

Comment:
Should we document the other 2 diagnostic setting variants as well?

Reply:
We can - we did with PE afterall. Should we then do this for all variants of all UDTs? Also also for e.g. Managed Identities?

Reply:
Added examples & tabs for **all** variants 

![image](https://github.com/user-attachments/assets/d12f8667-97db-487a-b297-0b8ec08a0fd1)